tests/test_utils.py

```diff
@@ -25,4 +26,4 @@ def test_diffs_one_different(self):
         for d1, d2 in permutations(dicts, 2):
             merged = d1.copy()
             merged.update(d2)
-            self.assertEqual(set(dict_diff(d1, merged)), set(d2.keys()))
+            self.assertEqual(set(dict_diff(d1, merged)), set(d2.keys()))
```

Comment:
Good effort trying to add a test. Please make sure to configure your editor so it adds a [trailing newline](https://stackoverflow.com/a/729795/2443886)! Omitting it only creates unnecessary diff noise.